packages/fetchai/connections/ledger/ledger_dispatcher.py

```diff
@@ -207,7 +207,7 @@ def get_transaction_receipt(
         while (
             transaction is None
             and attempts < self.MAX_ATTEMPTS
-            and self.connection_status.is_connected
+            and self.connection_state.get() == ConnectionStates.connected
```

Comment:
see above

Reply:
I pass state not connection, there is no is connected property